lib/internal/bootstrap/node.js

```diff
@@ -108,6 +108,12 @@ function startup() {
     process.exit = wrapped.exit;
   }
 
+  const experimentalPolicies = getOptionValue('--experimental-policy');
```

Comment:
This does not have to happen so early - from a glance I think it's fine to do this in `prepareUserCodeExecution` below, prior to loading `-r` modules.

Reply:
We just need to be sure to run this prior to *any* user code that could mutate core.

Reply:
We have certain expectations about the order the internal modules are loaded (e.g. C++ layer cannot create buffers after some code in the bootstrap process store the buffer prototype in C++, some modules or internalBinding incur side effects that need to happen in a certain order) so generally a module that loads other modules should happen as late as possible in order not to introduce subtle bugs. This is something that can happen much later and depend on a lot of modules not previously loaded by default.

Reply:
this is currently blocked a bit by other things trying to do things with the CJS system and it has to be loaded prior to that to avoid partial init problem of if it should be using policies or not

Reply:
Can you leave a TODO for me to investigate moving this to a point later? I see this has to happen early because of the dependency on CJSModule (which unfortunately is also depended on by all the ESM setup code), but this moves the first fs read to somewhere much earlier than before and leave us very little to snapshot in the future (with this, not even the Web API globals can be included in the snapshot).